src/Service.Host/client/src/components/Overbook.js

```diff
@@ -0,0 +1,132 @@
+    const handleFieldChange = (propertyName, newValue) => {
+        setState(d => ({ ...d, [propertyName]: newValue }));
+        setSaveDisabled(false);
+    };
+
+    const handleOverbookFieldChange = () => {
+        if (state.overbook !== 'Y') {
+            setState(d => ({ ...d, overbook: 'Y' }));
+        } else {
+            setState(d => ({ ...d, overbook: 'N' }));
+        }
+        setSaveDisabled(false);
+    };
+
+    return (
+        <Page history={history} homeUrl={config.appRoot}>
+            {overbookLoading ? (
+                <Loading />
+            ) : (
+                <Grid container spacing={3}>
+                    <Grid item xs={12}>
+                        <Typography variant="h3">Allow Overbook UT</Typography>
+                    </Grid>
+                    <Grid item xs={12}>
+                        <OnOffSwitch
+                            label="Overbook"
+                            value={state?.overbook === 'Y'}
+                            onChange={() => {
+                                handleOverbookFieldChange();
+                            }}
+                            propertyName="overbook"
+                        />
+                    </Grid>
+                    <Grid item xs={12}>
+                        <div className={classes.dialog}>
+                            <InputField
+                                value={state?.orderNumber}
+                                label="Order Number"
+                                propertyName="orderNumber"
+                                onChange={() => {}}
+                                disabled
+                            />
+                        </div>
+                    </Grid>
+                    <Grid item xs={12}>
+                        <InputField
+                            value={state?.overbookQty}
+                            label="Overbook Qty"
+                            type="number"
+                            propertyName="overbookQty"
+                            onChange={handleFieldChange}
+                            disabled={state.overbook !== 'Y'}
+                        />
+                    </Grid>
+                </Grid>
+            )}
+            <Grid item xs={6}>
+                <SaveBackCancelButtons
+                    saveDisabled={saveDisabled}
+                    saveClick={() => {
+                        setSaveDisabled(true);
+                        clearErrors();
+                        updatePurchaseOrder();
+                    }}
+                    cancelClick={() => {
+                        reduxDispatch({ type: 'initialise', payload: purchaseOrder });
+                    }}
+                    backClick={() => history.push('/purchasing/purchase-orders/allow-over-book/')}
+                />
+            </Grid>
+            <Grid item xs={12} />
+            <Grid item xs={12} />
+        </Page>
+    );
+}
+export default ViewPurchaseOrderForOverbook;
```

Comment:
Should probably add a 'Save Succesful' Message or display some kind of error message if not a success result